posthog/client.py

```diff
@@ -418,6 +407,16 @@ def substitute_params(query, params):
     containing code is only responsible for it's parameters, and we can
     avoid any potential param collisions.
     """
+    ch_client = SyncClient(
```

Comment:
It's so annoying that app we need this for is `substitute_params` 🤔  must be some way to not need this.